lib/trento_web/openapi/v1/schema/user.ex

```diff
@@ -0,0 +1,118 @@
+defmodule TrentoWeb.OpenApi.V1.Schema.User do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule UserCreationRequest do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "UserCreationRequest",
+      description: "Request body to create a user",
+      type: :object,
+      additionalProperties: false,
+      properties: %{
+        fullname: %Schema{type: :string, description: "User full name", nullable: false},
+        email: %Schema{type: :string, description: "User email", nullable: false, format: :email},
+        username: %Schema{type: :string, description: "User username", nullable: false},
+        enabled: %Schema{
```

Comment:
is `enabled` here the opposite of `locked`?
It is maybe a bit strange to use `enabled `term in the controller and `locked` in the "backend/schemas".
But i undestand the terms anyway

Reply:
Yeah it's true, on the frontend/api we use enabled so I stick with it but we can change